holidays/countries/__init__.py

```
@@ -159,6 +159,7 @@
 from .suriname import Suriname, SR, SUR
 from .sweden import Sweden, SE, SWE
 from .switzerland import Switzerland, CH, CHE
+from .syrian_arab_republic import SyrianArabRepublic , SY  
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)